### PR TITLE
Upgraded the 3rd party library

### PR DESCRIPTION
**Pull request by victorray84**
I've upgraded the 3rd party library.

### CODE REVIEW

**Comment by victorray84** [comment]
I've upgraded the third party library. Please can anybody fix the merge conflicts.

**Comment by ualch9** [comment]
I think you will need to squash your commits before this can get merged (https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history)

**Comment by aaronbrethorst** [comment]
Hey @victorray84 - thanks for your interest in helping out. As a general rule, I think it's best for new contributors to start with one of the issues marked with the "Your First PR" label, as those tend to not have unexpected complications or dependencies. (See https://github.com/OneBusAway/onebusaway-iphone/issues?q=is%3Aissue+is%3Aopen+label%3A%22Your+First+PR%22)

A few things:

1. You'll need to make sure that this PR merges cleanly in order for us to start considering it.
2. I notice that Apptentive is not one of the libraries you upgraded. They've made their library work correctly with Carthage, so as long as 3rd party libraries are being updated, that should be one that gets updated.
3. Have you verified that this PR works correctly?

**Comment by victorray84** [comment]
@aaronbrethorst Thanks for your quick response. Sure, I will fix "Your First PR" issue soon.

**Comment by victorray84** [comment]
@aaronbrethorst can you tell me why it's getting failed?

**Comment by aaronbrethorst** [comment]
Not offhand. Have you tried running tests locally?

**Comment by aaronbrethorst** [comment]
You need to run tests and make sure they pass. And also make sure that you rebase and squash your commits. thanks!

File: Cartfile.resolved [hunk 1]
```diff
@@ -2,16 +2,15 @@ github "aaronbrethorst/CocoaLumberjack" "845ff8043fb6b902cc0e09d34ecf28bdf6d1cee
 github "dzenbot/DZNEmptyDataSet" "v1.8.1"
 github "cbpowell/MarqueeLabel" "3.0.1"
 github "SnapKit/Masonry" "v1.0.2"
-github "OneSignal/OneSignal-iOS-SDK" "2.3.5"
```
Comment (aaronbrethorst) [inline Cartfile.resolved]: This needs to be retained.

**Comment by victorray84** [comment]
@aaronbrethorst Please check the pull request. I've added the OneSignal sdk to cartfile.

**Comment by aaronbrethorst** [comment]
@victorray84 per my earlier comment, you need to rebase and squash your commits. Let me know if you need a pointer to a resource on how to do this.

**Comment by victorray84** [comment]
@aaronbrethorst Please check my pull request. I've done squash and rebase of my commits. I've followed the following tutorial https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request for squash and rebase. Please let me know whether I'm doing right or not.

**Comment by aaronbrethorst** [comment]
This PR currently says that there are 22 commits associated with it. At the end of a successful rebase and squash, you should be left with 1 commit.

Steps generally associated with a successful rebase and squash:

1. Make sure you have your upstream remote set to https://github.com/OneBusAway/onebusaway-iphone/
2. Make sure you're working on a feature branch and not `develop`
2. `git fetch upstream`
3. `git rebase -i upstream/develop`
4. (squash your commits)
5. `git push origin <YOUR BRANCH NAME> -f`

**Pull request closed without merging**